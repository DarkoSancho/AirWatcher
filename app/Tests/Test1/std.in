1
government
5
2
