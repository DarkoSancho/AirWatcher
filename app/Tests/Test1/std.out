Welcome to AirWatcher App!

Menu:
1. Sign In
2. Exit
Select your choice: 
Enter user login: 
AirWatcher App -  Welcome to AirWatcher ! Let's get started.

AirWatcher App - Government Agency Menu 

1) - Calculate mean air quality over area.
2) - Show statistics for a specific sensor. 
3) - Score and rank sensors by similarity.
4) - Performance tests.
5) - Sign Out.

Select a functionality by choosing the corresponding number:


AirWatcher App- You signed out successfully !

Menu:
1. Sign In
2. Exit
Select your choice: 
Exiting...
